CMakeLists.txt

```diff
@@ -37,21 +37,21 @@ if (NOT DEFINED PROTO_SEARCH_PATH)
   set(PROTO_SEARCH_PATH ${CMAKE_CURRENT_SOURCE_DIR})
 endif()
 
-# protoc flags
+# PROTOBUF_PROTOC_EXECUTABLE flags
 set(PROTO_CFLAGS)
-list(APPEND PROTOC_FLAGS "--proto_path=${PROTO_SEARCH_PATH}")
-list(APPEND PROTOC_FLAGS "--${GEN_LANG}_out=${CMAKE_CURRENT_BINARY_DIR}") 
+list(APPEND PROTOBUF_PROTOC_EXECUTABLE_FLAGS "--proto_path=${PROTO_SEARCH_PATH}")
+list(APPEND PROTOBUF_PROTOC_EXECUTABLE_FLAGS "--${GEN_LANG}_out=${CMAKE_CURRENT_BINARY_DIR}") 
 
 # Build grpc if the plugin is found
 if (EXISTS ${GRPC_PLUGIN})
-  list(APPEND PROTOC_FLAGS "--plugin=protoc-gen-grpc=${GRPC_PLUGIN}")
-  list(APPEND PROTOC_FLAGS "--grpc_out=${CMAKE_CURRENT_BINARY_DIR}") 
+  list(APPEND PROTOBUF_PROTOC_EXECUTABLE_FLAGS "--plugin=PROTOBUF_PROTOC_EXECUTABLE-gen-grpc=${GRPC_PLUGIN}")
```

Comment:
I dunno, but can you please double check this replacement after "--plugin". I think this may not apply.